integration-tests/docker/tls/generate-server-certs-and-keystores.sh

```diff
@@ -17,6 +17,12 @@
 
 cd /tls
 
+FILE_CHECK_IF_RAN=/tls/server.key
+if [ -f "$FILE_CHECK_IF_RAN" ]; then
+  echo "Script was ran already. Skip running again."
```

Comment:
nit: No need to change if everything else looks good. If I saw the log line as is, it's a little ambiguous - which script? what's the impact of skipping running again?
```suggestion
  echo "Using existing tls keys since /tls/server.key exists - skipping generation of all certs. To generate certs, delete this file"
```

Reply:
Done